app/models/user.rb

```diff
@@ -49,6 +49,7 @@ class User < ApplicationRecord
   acts_as_followable
   acts_as_follower
 
+  has_one :profile, dependent: :destroy
```

Comment:
Apart from profiles being inherently tied to users, this was also added to satisfy current specs and the foreign key constraint.